src/components/Events/Event.vue

```diff
@@ -8,7 +8,7 @@
         <p>{{ this.description }}</p>
       </div>
       <div class="event-btns">
-          <button class="event-btn">Learn More</button>
+          <button class="event-btn">Learn More</button><br />
```

Comment:
Can you do this with css styling as opposed to <br>?

Reply:
Didn't realize that was still in there, I'll fix it